packages/etherlime-lib/deployed-contract/deployed-contract-wrapper.js

```diff
@@ -26,6 +27,11 @@ class DeployedContractWrapper {
 		this.estimate = this.contract.estimate
 		this.functions = this.contract.functions
 		this.filters = this.contract.filters
+		this.utils = {
+            getBalance: async function() {
+                return await self.contract.provider.getBalance(self.contractAddress)
```

Comment:
I think that you can rewrite the whole of this as :
```
getBalance: self.provider.getBalance(self.contractAddress)
```

No need for the async and await as getBalance is already a promise